### PR TITLE
Signals: correctly return zero values

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: test/unit/utils/test-signals.js [hunk 1]
```diff
@@ -176,3 +176,18 @@ describes.sandboxed('Signals', {}, () => {
...
+  let signals;
+
+  beforeEach(() => {
+    clock = sandbox.useFakeTimers();
+    signals = new Signals();
+  });
+
+  it('should register signal without promise', () => {
+    signals.signal('sig');
+    expect(signals.get('sig')).to.equal(0);
```
Comment (lannka) [inline test/unit/utils/test-signals.js]: pls add a comment explain why this is 0 (because fakeTImer)

File: src/utils/signals.js [hunk 2]
```diff
@@ -52,7 +52,8 @@ export class Signals {
    * @return {number|!Error|null}
    */
   get(name) {
-    return this.map_[name] || null;
+    const v = this.map_[name];
+    return v === 0 ? v : v || null;
```
Comment (jridgewell) [inline src/utils/signals.js]:
```suggestion
    return v == null ? null : v;
```

To match with the [nullish coalescing operator](https://babeljs.io/repl/#?babili=false&browsers=Safari%20>%209&build=&builtIns=false&spec=false&loose=false&code_lz=MYewdgzgLgBAbjAvDADAbgFAIPzZmAVwBsi0g&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015-loose&prettier=true&targets=&version=7.5.5&externalPlugins=%40babel%2Fplugin-proposal-nullish-coalescing-operator%407.4.4): `v ?? null`

File: test/unit/utils/test-signals.js (hunk 1 above)
Comment (dvoytenko) [reply test/unit/utils/test-signals.js]: Done!

File: src/utils/signals.js (hunk 2 above)
Comment (dvoytenko) [reply src/utils/signals.js]: Done!

**Pull request merged**